x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -482,6 +482,7 @@ export const fillDefineNewTermsRuleAndContinue = (rule: NewTermsRuleCreateProps)
   cy.get(NEW_TERMS_INPUT_AREA).find(INPUT).type(rule.new_terms_fields[0], { delay: 35 });
 
   cy.get(EUI_FILTER_SELECT_ITEM).click();
+  // eslint-disable-next-line cypress/unsafe-to-chain-command
   cy.focused().type('{esc}'); // Close combobox dropdown so next inputs can be interacted with
```

Comment:
I'm not a Cypress expert, so I'm not sure if there's a way to rewrite the code without using chaining so that we don't have to ignore this ESLint error. Feel free to suggest a fix

Reply:
I think it's best to just keep the ignores. This will probably get fixed soon an when we upgrade it will let us know that the ignores are no longer needed